src/components/controls/color-by.js

```diff
@@ -177,13 +177,14 @@ class ColorBy extends React.Component {
 
   render() {
     const styles = this.getStyles();
+
     const colorOptions = Object.keys(this.props.colorings)
       .map((key) => ({value: key, label: this.props.colorings[key].title}));
+
     return (
-      <div style={styles.base}>
+      <div style={styles.base} id="selectColorBy">
```

Comment:
Lifted `id` up one level, since it's used in the test below for assertion. I believe `id="selectColorBy"` wasn't rendered in the DOM, since `<Select />` ignores it